packages/components/src/components/select-native/select-native.scss

```diff
@@ -55,6 +55,10 @@
         justify-content: flex-start;
         padding-left: 1.2rem;
 
+        @include tablet-screen {
+            max-width: 568rem;
```

Comment:
why 568 rem? it should be 56.8rem. isnt?

Reply:
thank you for pointing this out 🙏